crates/ink/macro/src/lib.rs

```diff
@@ -677,6 +680,15 @@ pub fn trait_definition(attr: TokenStream, item: TokenStream) -> TokenStream {
 ///     #[ink(topic)]
 ///     pub topic: [u8; 32],
 /// }
+/// // Setting `signature_topic = <hash_string>` specifies custom signature topic.
```

Comment:
```suggestion
/// // Setting `signature_topic = <hex_string>` specifies custom signature topic.
```
Ah maybe you mean hex and not hash string?

Reply:
Yes, I meant hex